bftengine/src/bftengine/ReplicaImp.cpp

```diff
@@ -3942,6 +4047,7 @@ ReplicaImp::ReplicaImp(const LoadedReplicaData &ld,
   }
 
   internalThreadPool.start(config_.getsizeOfInternalThreadPool());
+  postExecThread_.start(1);  // TODO(LG) should be configurable;
```

Comment:
MUST be configurable :-)

Reply:
As explained this comment was my mistake - it **must** be only with size 1.